model.go

```diff
@@ -6,18 +6,31 @@ import (
 	"os"
 )
 
-type Param struct {
+type Model []Layer
+
+type Layer struct {
```

Comment:
__[golint]__
* [golint] exported type Layer should have comment or be unexported